src/commands/git/branch.ts

```diff
@@ -251,7 +264,12 @@ export class BranchGitCommand extends QuickCommand {
 				state.subcommand,
 			);
 
-			if (state.counter < 2 || state.repo == null || typeof state.repo === 'string') {
+			if (
+				state.counter < 2 ||
+				state.repo == null ||
+				typeof state.repo === 'string' ||
+				castCreateState(state)?.suggestRepoOnly
```

Comment:
Rather than using a casting function, what do you think about just asserting first and then using the property:
```suggestion
				(isCreateState(state) && state.suggestRepoOnly)
```

Reply:
fixed